hfc/fabric_network/wallet.py

```diff
@@ -28,6 +30,18 @@ def remove(self, enrollment_id):
         if dirpath.exists() and dirpath.is_dir():
             shutil.rmtree(dirpath)
 
+    def get_user(self, enrollment_id, org, msp_id, state_store=None):
```

Comment:
this creates a user instance rather than getting a user instance, so I think it should not be called `get_user` method.

Reply:
The function is similar but has been implemented in a class called Gateway.
Will change that!